sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -435,6 +435,11 @@ object SQLConf {
     defaultValue = Some(true),
     doc = "When false, we will treat bucketed table as normal table")
 
+  val ORDER_BY_ORDINAL = booleanConf("spark.sql.orderByOrdinal",
+    defaultValue = Some(true),
```

Comment:
We are facing two options. Not sure which one is better. For RDBMS users, the default is true.